src/elastic/azext_elastic/aaz/latest/elastic/monitor/_associate_traffic_filter.py

```diff
@@ -22,16 +22,17 @@ class AssociateTrafficFilter(AAZCommand):
     """
 
     _aaz_info = {
-        "version": "2023-02-01-preview",
+        "version": "2024-06-15-preview",
         "resources": [
-            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.elastic/monitors/{}/associatetrafficfilter", "2023-02-01-preview"],
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.elastic/monitors/{}/associatetrafficfilter", "2024-06-15-preview"],
         ]
     }
 
+    AZ_SUPPORT_NO_WAIT = True
+
     def _handler(self, command_args):
         super()._handler(command_args)
-        self._execute_operations()
-        return None
+        return self.build_lro_poller(self._execute_operations, None)
```

Comment:
It seems this api become long running operation. Could you verify if it's expected.

Reply:
Changed.